{{ cookiecutter.project_slug }}/{{ cookiecutter.python_slug }}/reader_metadata.py

```diff
@@ -19,6 +25,9 @@ def get_supported_extensions() -> List[str]:
 
     @staticmethod
     def get_reader() -> bioio_base.reader.Reader:
+        """
+        Return reader this plugin.
```

Comment:
`Return the reader this plugin represents` maybe?

Reply:
resolved in [e01bd2c](https://github.com/bioio-devs/cookiecutter-bioio-reader/pull/3/commits/e01bd2c1c16950ee139843c4df8a91bff88ef8b0)